test/controllers/still.required.controllers.spec.unit.ts

```diff
@@ -34,4 +33,26 @@ describe("still required validation tests", () => {
     expect(response.text).toContain("Tell us if THE GIRLS DAY SCHOOL TRUST is required");
     expect(response.text).toContain(COMPANY_REQUIRED_NOT_SELECTED);
   });
+
+  it("should update session when yes is selected", async () => {
+    const session: Session = Session.newInstance();
+    const response = await request(app)
+        .post("/promise-to-file/company/00006400/still-required")
+        .set("Referer", "/")
+        .set("Cookie", [`${COOKIE_NAME}=123`])
+        .send({stillRequired: true});
+
+    expect(response.status).toEqual(302);
```

Comment:
missing the session value check

Reply:
I know - not sure how to check this in a jest unit test we are mocking the session.

Reply:
ok i'll have a think